components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -3177,19 +3223,31 @@ public void deleteIdPByResourceId(String resourceId, int tenantId, String tenant
             throws IdentityProviderManagementException {
 
         Connection dbConnection = IdentityDatabaseUtil.getDBConnection();
+        String idPName = "";
```

Comment:
```suggestion
```

Reply:
Since we need to mention the idpName in the error message, we need this outside the try block